components/rhel/misc/hyperv/rhel-ose-hyperv/README.rst

```diff
@@ -0,0 +1,61 @@
+Vagrantfile to setup  OpenShift Enterprise
+==========================================
+
+This Vagrantfile sets up the CDK for development with OpenShift. This Vagrantfile provisions an instance of `OpenShift Enterprise <https://www.openshift.com/enterprise/whats-new.html>`_.
+
+**Note:** This Vagrantfile expects an external type virtual switch for HyperV. Please add a new external type virtual network switch to the HyperV setup if you do not have it.
+
+
+QuickStart
+----------
+
+1. Get latest CDK box (Using Developer Subscription).
+
+2. Create a directory for the Vagrant Box
+
+  ``$ mkdir directory && cd directory``
+
+3. Download this Vagrantfile in same directory.
+
+4. Start the Vagrant Box
+
+  ``$ vagrant up``
+  
+  **Note:** HyperV assigns a DHCP IP address to the CDK Vagrant Box. The required IP and login information will be displayed at the end of the `vagrant up` process.
+
+5. Proceed with using the CDK.
+
+  If you plan to use OpenShift from the host, you may wish to download ``oc`` for your platform from an `OpenShift Enterprise Release <https://access.redhat.com/downloads/content/290>`_.
+
+  You may wish to verify that OpenShift is setup using the ``oc`` CLI as follows:
+
+  1. Login to OpenShift using *oc*
+
+    ``$ oc login <ip>:<port>``
```

Comment:
how does user know what is IP:PORT to be used ?

Can we hint here to use VSM ?